lib/createCookieEncryptor.js

```diff
@@ -0,0 +1,73 @@
+import base64url from 'base64url'
+
+const KEY_ALGORITHM = 'AES-GCM'
+
+const toHexString = bytes =>
+  bytes.reduce((str, byte) => str + byte.toString(16).padStart(2, '0'), '')
+
+const fromHexString = hexString =>
+  new Uint8Array(hexString.match(/.{1,2}/g).map(byte => parseInt(byte, 16)))
+
+export const createCookieEncryptor = (cookieName, encryptionKey) => {
```

Comment:
I think we should provide these params as an options object, it will make this more extensible in the future.